src/helpers/format.ts

```diff
@@ -244,19 +271,32 @@ export function applyDateTimeFormat(value: number, format: Format): FormattedVal
 }
 
 function formatJSDate(jsDate: Date, format: Format): FormattedValue {
-  const sep = format.match(/\/|-|\s/)![0];
-  const parts = format.split(sep);
+  const sep = format.match(/\/|-|\s/)?.[0];
+  const parts = sep ? format.split(sep) : [format];
   return parts
     .map((p) => {
       switch (p) {
         case "d":
           return jsDate.getDate();
         case "dd":
           return jsDate.getDate().toString().padStart(2, "0");
+        case "ddd":
+          return DAYS[jsDate.getDay()].slice(0, 3);
+        case "dddd":
+          return DAYS[jsDate.getDay()];
         case "m":
           return jsDate.getMonth() + 1;
         case "mm":
           return String(jsDate.getMonth() + 1).padStart(2, "0");
+        case "mmm":
+          return MONTHS[jsDate.getMonth()].slice(0, 3);
+        case "mmmm":
+          return MONTHS[jsDate.getMonth()];
+        case "mmmmm":
+          return MONTHS[jsDate.getMonth()].slice(0, 1);
+        case "yy":
+          const fullYear = String(jsDate.getFullYear()).replace("-", "").padStart(2, "0");
```

Comment:
Don't see the tests for this very particular case. :p

Reply:
the test can be written with -692132  on which to apply the date format --> give the year 1/1/0005

Reply:
Done :)